### ISSUE TITLE
Cannot reproduce the result of run_squad 1.1

### ISSUE THREAD

**Issue by hmt2014** [huggingface/pytorch-pretrained-BERT#168]
I train 5 epochs with learning rate 5e-5, but my evaluation result is {'exact_match': 32.04351939451277, 'f1': 36.53574674513405}.
What is the problem?

**Comment by zhaoguangxiang** [comment]
I can reproduce the results, learning rate is 3e-5 , epoch is 2.0

**Comment by zhaoguangxiang** [comment]
by using fp16, the f1 is 90.8

**Comment by hmt2014** [comment]
> by using fp16, the f1 is 90.8

So the key is set fp16 is True?

**Comment by thomwolf** [comment]
Yes, please use the command line example indicated [here](https://github.com/huggingface/pytorch-pretrained-BERT#squad) in the readme for SQuAD.
